### PR TITLE
Fixed a grammar problem and made the paragraph less confusing.

### PR DESCRIPTION
**Pull request by GitHubPang**
Similar to commit e799ea068592d2cad1037ca9c2e447557cc5ff18.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@GitHubPang Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=992)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=992) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by GitHubPang** [comment]
@pivotal-issuemaster This is an Obvious Fix

**Comment by pivotal-issuemaster** [comment]
@GitHubPang This Pull Request contains an obvious fix. Signing the Contributor License Agreement is not necessary.

File: site/tutorials/tutorial-one-php.md [hunk 1]
```diff
@@ -145,9 +145,9 @@ class](https://github.com/rabbitmq/rabbitmq-tutorials/blob/master/php/send.php).
 
 ### Receiving
 
-That's it for our publisher.  Our receiver listening for messages from
+That's it for our publisher.  Our receiver listens for messages from
 RabbitMQ, so unlike the publisher which publishes a single message, we'll
-keep it running to listen for messages and print them out.
+keep the receiver running to listen for messages and print them out.
```
Comment (michaelklishin) [inline site/tutorials/tutorial-one-php.md:150]: Why does this say "the receiver" while other ports say "the consumer"?

Comment (GitHubPang) [reply site/tutorials/tutorial-one-php.md:150]: Because it says "Our receiver" in the same paragraph (line 148).

Comment (michaelklishin) [reply site/tutorials/tutorial-one-php.md:150]: Fair enough!

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**